manifests/m/M-FilesCorporation/M-FilesOnline/23.4.12528.8/M-FilesCorporation.M-FilesOnline.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: M-FilesCorporation.M-FilesOnline
+PackageVersion: 23.4.12528.8
+DefaultLocale: eng
```

```suggestion
DefaultLocale: en-US
```
Please use the BCP-47 language codes.